core/edit/edit.talon

```diff
@@ -161,20 +138,17 @@ cut all:
 #     edit.select_all()
 #     edit.cut()
 
-cut word:
-    edit.select_word()
-    edit.cut()
-
-cut word left: user.cut_word_left()
-
-cut word right: user.cut_word_right()
-
-cut line: user.cut_line()
-
-(pace | paste) all:
-    edit.select_all()
-    edit.paste()
+# Paste
+(pace | paste) all: user.paste_all()
+(pace | paste) line: user.paste_line()
+(pace | paste) line start: user.paste_line_start()
+(pace | paste) line end: user.paste_line_end()
+(pace | paste) word: user.paste_word()
 
-# duplication
+# Duplication
 clone that: edit.selection_clone()
 clone line: edit.line_clone()
+
+# Insert new line
+new line above: edit.line_insert_up()
+new line below: edit.line_insert_down()
```

Comment:
We already have `slap` for this. I don't think we need another one.

https://github.com/knausj85/knausj_talon/blob/2e9e03ddb559c782e65650120a9d91079b6dc643/core/edit/standard.talon#L22

Reply:
I think it's fine to have a duplicate since otherwise we'll have `new line above` without `new line below` which would be very confusing.

Reply:
Fair. Maybe worth considering what should go in `standard.talon` as opposed to `edit.talon` then.

Reply:
I think we should remove `standard.talon` and move the content to `edit.talon`.